CorsixTH/Lua/dialogs/hire_staff.lua

```diff
@@ -169,7 +169,7 @@ function UIHireStaff:draw(canvas, x, y)
     if not profile then
       return
     end
-    font:draw(canvas, profile.name, x + 79, y + 21)
+    font:draw(canvas, profile.initial .. ". " .. profile.name, x + 79, y + 21)
```

Comment:
As you can see in the diff, you have this sequence everywhere.

This implies it may be a good idea to move that concatenation into the `profile` class.
I think that is defensible, the profile manages all kinds of personal properties, so the full name with initials fits in it naturally.

Reply:
Putting the two parts together any earlier breaks the last name translation.

Reply:
So you have to do this when the name is requested.
Fair enough.

what about making a new function
``` lua
function StaffProfile:getFulleName()
  return self.initial .. ". " .. self.name
end
```
and use that?

That eliminates the clutter everywhere too.